northstar_tests/tests/tests.rs

```diff
@@ -214,16 +220,24 @@ test!(crashing_container, {
 
     for _ in 0..10 {
         runtime.test_cmds("crash").await;
+
         runtime.start(TEST_CONTAINER).await?;
         runtime
             .assume_notification(
                 |n| {
-                    n == &Notification::Exit {
+                    let exited = Notification::Exit {
                         container: TEST_CONTAINER.try_into().unwrap(),
                         status: ExitStatus::Exit(254),
-                    }
+                    };
+
+                    let aborted = Notification::Exit {
+                        container: TEST_CONTAINER.try_into().unwrap(),
+                        status: ExitStatus::Signaled(6),
+                    };
+
+                    n == &exited || n == &aborted
```

Comment:
Shouldn't the test container always exit with the same result?

Reply:
Yes, you are right. It is now expected to exit via an abort signal.